src/board/Cell.jsx

```diff
@@ -1,9 +1,26 @@
+import React, { useState } from "react";
 import "./Cell.css";
 import PropTypes from "prop-types";
+import { isValidMove } from "../validation/validateMove";
+
+function Cell({ cellSpot, handleClick, gameStatus }) {
+  console.log(cellSpot);
```

Comment:
did you mean to leave this comment in? always good to review your changes before making any commits :)